### PR TITLE
[RLlib] APPO enhancements (new API stack) vol 02: Cleanup loss function, add GAE-lambda to vtrace, make rho-clip configurable.

### PR DESCRIPTION
**Pull request by sven1977**
APPO enhancements (new API stack) vol 02:
* Cleanup loss function (cleaner , more consistent namings)
* add GAE-lambda to vtrace
* make rho-clip configurable

## Checks

- [x] I've signed off every commit(by using the -s flag, i.e., `git commit -s`) in this PR.
- [x] I've run `scripts/format.sh` to lint the changes in this PR.
- [ ] I've included any doc changes needed for https://docs.ray.io/en/master/.
    - [ ] I've added any new APIs to the API Reference. For example, if I added a 
           method in Tune, I've added it in `doc/source/tune/api/` under the 
           corresponding `.rst` file.
- [ ] I've made sure the tests are passing. Note that there might be a few flaky tests, see the recent failures at https://flakey-tests.ray.io/
- Testing Strategy
   - [ ] Unit tests
   - [ ] Release tests
   - [ ] This PR is not tested :(

### CODE REVIEW

**Review by simonsays1980 — APPROVED** [review]
LGTM. Everything gets cleaned up now. This looks really good. Do we have already first training results?

File: rllib/algorithms/appo/torch/appo_torch_learner.py [hunk 1]
```diff
@@ -60,45 +71,49 @@ def compute_loss_for_module(
...
+        current_actions_logp = current_action_dist.logp(batch[Columns.ACTIONS])
+        current_actions_logp_time_major = make_time_major(
+            current_actions_logp,
             trajectory_len=rollout_frag_or_episode_len,
             recurrent_seq_len=recurrent_seq_len,
         )
+
+        # Target policy.
+        target_action_dist = action_dist_cls_train.from_logits(
+            module.forward_target(batch)[TARGET_ACTION_DIST_LOGITS_KEY]
```
[the remark this answers is not in the document]
Comment (sven1977) [reply rllib/algorithms/appo/torch/appo_torch_learner.py:88]: You are right, but it does seem to run fine in the CI (multi-GPU). 🤷‍♂️
